Cargo.toml

```diff
@@ -8,6 +8,9 @@ description = "Simple work-stealing parallelism for Rust"
 license = "Apache-2.0/MIT"
 repository = "https://github.com/nikomatsakis/rayon"
 documentation = "https://docs.rs/rayon/"
+readme = "README.md"
+keywords = ["parallel", "thread", "concurrency", "join", "performance"]
+categories = ["concurrency"]
```

Comment:
Can you also add these attributes to rayon-core and rayon-futures?

Reply:
I'm not sure if readme attribute is appropriate.  I added '..\readme.md'.  Not sure if that is helpful or if the other 'crates' need their own readmes.

Reply:
I have no idea if crates.io will work with `../README.md` -- I doubt it though, because it's probably looking at the individual uploaded crates, not the repository.

We do have a readme in rayon-futures, but not rayon-core.  The latter could just be a simple placeholder, saying something like "Core APIs for Rayon; see the rayon crate for more details."